biomage/unstage/unstage.py

```diff
@@ -31,76 +202,85 @@ def check_if_exists(org, sandboxid):
     default="biomage-ltd",
     help="The GitHub organization to perform the operation in.",
 )
-def unstage(token, org, sandboxid):
+def unstage(token, org, sandbox_id):
     """
     Removes a custom staging environment.
     """
 
-    if not check_if_exists(org, sandboxid):
+    # Read configuration
+    config = None
+    with open("config.yaml") as config_file:
+        config = list(yaml.load_all(config_file, Loader=yaml.SafeLoader))[0]
+
+    if check_if_exists(org, sandbox_id):
```

Comment:
as said earlier, try to name function by the return value and intended use. `if check_if_exists` is not very clear (which resource is it checking? if if exists what does it check? what does it return) it is clearer `if sandbox_exists` -> if sandbox_exists return true it's clear the sandbox exists, and when used with an `if` it can be read naturally

Reply:
renamed to `check_if_sandbox_exists`